### DISCUSSION TITLE
V3 machine screw fit issues

### DISCUSSION THREAD

**Discussion by bannus** [doudar/SmartSpin2k#479] [Q&A] [answered]
Working on a V3 build, and run into some minor issues with the machine screws. The 5/16" x 1.5" hex head bolt fits great for the axle. However, the diameter of the M5 x 30mm cap screw is a very tight fit for the holes in the case and the arm. I can get it to work, but it requires a good deal of force to thread it into the material (I'm using PLA, if it matters). Is this intentional? I noticed that the hole for the arm is faceted, and has a minimum diameter of 4.6mm and a maximum of 5.0mm. The holes on the case are circular, and are exactly 5mm.

![image](https://github.com/doudar/SmartSpin2k/assets/1330640/b593d6e3-6d8b-433d-bc82-28f2f1c3a1cd)
![image](https://github.com/doudar/SmartSpin2k/assets/1330640/32ac5f79-e3a8-4776-84b2-0234ae889ac1)

The old [adjustable arm](https://github.com/doudar/SmartSpin2k/blob/5a7afb55ce42b08f1bad0423d687dd05dafc5379/Hardware/Common%20Assets/Arm/schwinn%20ic4.stl) had a 5.05mm diameter circular hole.

Also, the bike mount instructions call for an M4 x 30mm cap screw and nut that aren't mentioned in the hardware list. However, I was able to get a M5 x 30mm in (with some force required), and the M5 nut is a perfect fit for the part, whereas an M4 nut is quite loose. The model for the IC4 mount has 5.1mm diameter circles, so I'm assuming the instructions are supposed to say M5 here.

**Comment by eMadman** [comment, marked as answer]
It is intended to be an M5 screw on this hole - the fixed length ones are a bit tighter in comparison.  I can see having a harder time threading the screw into it with PLA.  I'm usually using ABS which is a lot softer on that front.  

I think OpenSCAD outputs the circles with a hexagonal shape.  That's pretty neat.  You can use the configurator on thingiverse to make the bolt diameter a touch wider: https://www.thingiverse.com/thing:5732525

I'll work on a PR to make this change in the next few days.

**Reply by eMadman** [reply]
@bannus - it was an easy fix in the openscad.  Check the pull request here: https://github.com/doudar/SmartSpin2k/pull/480

I think you can grab the STL from the list of files changed

**Reply by bannus** [reply]
Great, thanks for the fix! I did manage to make it work with the print from the existing STL, but it's pretty tight. If I have issues, I'll print a new arm.

How about the screw for the mount? Is that one supposed to be an M5 as well? If so, the PDF instructions could use an update, both to list the additional screw in the hardware list on page 4 as well as the step on page 13.

**Reply by doudar** [reply]
Hmmm... There is a M4 x 30 screw in the .pdf. 

@eMadman , I'll go verify this actual size quick unless you know off the top of your head.

![image](https://github.com/doudar/SmartSpin2k/assets/17362216/f7c93a24-f157-499c-9fad-c312cbcb95e6)

**Reply by bannus** [reply]
FWIW I did use an M5 for my IC4 build. The hole for the IC4 mount is 5.1mm and the cutout for the nut is 8.4mm, which seems like it was made for an M5x0.8 nut. The other mounts could be made for the M4 size though.

Edit: just checked the Peloton mount, and I see the same dimensions on the hole and the cutout as the IC4 mount.

![image](https://github.com/doudar/SmartSpin2k/assets/1330640/b770c8a3-6c81-405f-b5cd-08284b23f4d8)

**Reply by doudar** [reply]
Looking at mine, the m4 is a typo.  I'll fix the .pdf. Thanks for finding it!!
